connectors/opa/lib/evaluate_opa.go

```diff
@@ -75,5 +89,15 @@ func EvaluatePoliciesOnInput(inputMap map[string]interface{}, opaServerURL strin
 	log.Println("responsestring data")
 	log.Println(string(data))
 
-	return string(data), nil
+	currentData := string(data)
+	decisionid, flag := doesOpaHaveUserPoliciesLoaded(string(data))
```

Comment:
the json parser expects []byte, why cast to string and back to []byte?

Reply:
Fixed. Please check. Thanks.